extension.ts

```diff
@@ -0,0 +1,65 @@
+import { Facet, Extension, RangeSet } from "@codemirror/state"
+import { WidgetType, EditorView, Decoration, ViewPlugin, DecorationSet, ViewUpdate, MatchDecorator } from "@codemirror/view"
+
+interface MorgenPluginSettings {
+	decorateIDs: 'show' | 'hide' | 'replace_with_emoji'
+}
+
+const DEFAULT_SETTINGS: MorgenPluginSettings = {
+	decorateIDs: 'show'
+}
```

Comment:
Do we want to default show the id?

Reply:
Yeah I think it's best to avoid anything unexpected and allow them to decide between the emoji and hiding it completely.